### PR TITLE
kops: Add full suite (preparing to move to kops testing entirely)

### PR DESCRIPTION
**Pull request by zmerlynn**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/965)

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-kops-aws.yaml [hunk 1]
```diff
@@ -59,5 +59,11 @@
             job-env: |
                 export E2E_NAME="e2e-kops-aws-updown"
                 export GINKGO_TEST_ARGS="--ginkgo.focus=\[k8s.io\]\sNetworking.*\[Conformance\]"
+        - 'aws':  # kubernetes-e2e-kops-aws
+            description: 'Run e2e tests on AWS deployed via kops using the latest successful Kubernetes build.'
+            timeout: 240
+            job-env: |
+                export E2E_NAME="e2e-kops-aws"
+                export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[HPA\]|NodeProblemDetector"
```
Comment (spxtr) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-kops-aws.yaml]: I have a strong repulsion from skipping anything that isn't a tag.

Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-kops-aws.yaml]: This is actually copied over from the AWS tests, but I forgot to include the comment. I'll push that now.

Comment (zmerlynn) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-kops-aws.yaml]: Done, PTAL

**Comment by spxtr** [comment]
As long as that `NodeProblemDetector` skip is temporary...
/lgtm

**Comment by zmerlynn** [comment]
@spxtr: I'm investigating it again now..

**Pull request merged**